stock_eggs/source-engine/egg-garrys-mod.json

```diff
@@ -99,6 +99,15 @@
             "user_viewable": true,
             "user_editable": true,
             "rules": "required|boolean"
+        },
+        {
+            "name": "Beta",
+            "description": "Determines which server version should be used. | NONE = stable release | x86-64 = 64Bit beta.\r\nIt is necessary to reinstall the server when changing the version here.",
+            "env_variable": "BETA",
+            "default_value": "NONE",
```

Comment:
Is `NONE` a proper value for this to not fail?

Reply:
I re-read the desc you put in and it looks like it is.

Reply:
Yeah, `NONE` is the value for the stable version of the game.